src/loaders/lottie/tvgLottieModel.h

```diff
@@ -464,10 +464,10 @@ struct LottieGradient : LottieObject
         if (!colorStops.populated) {
             if (colorStops.frames) {
                 for (auto v = colorStops.frames->begin(); v < colorStops.frames->end(); ++v) {
-                    colorStops.count = populate(v->value);
```

Comment:
why don't you? 
![image](https://github.com/thorvg/thorvg/assets/3711518/4eaac8a6-9fc6-41b2-aedf-b394726ac82f)